### PR TITLE
1.20.1 should use JDK 17 to compile

### PR DESCRIPTION
**Pull request by Thodor12**
Review please

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
   release:
     uses: ldtteam/operapublicacreator/.github/workflows/gradle.publish.yaml@main
     with:
-      java: 21
+      java: 17
     secrets:
       CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
-      GRADLE_ENCRYPTION_KEY: ${{ secrets.GRADLE_ENCRYPTION_KEY }}
\ No newline at end of file
+      GRADLE_ENCRYPTION_KEY: ${{ secrets.GRADLE_ENCRYPTION_KEY }}
```
Comment (Raycoms) [inline .github/workflows/build.yaml:30]: readd newline please

Comment (Thodor12) [reply .github/workflows/build.yaml:30]: #blame-github

Comment (Nightenom) [reply .github/workflows/build.yaml:30]: he added them :D (they werent there before)

Comment (Raycoms) [reply .github/workflows/build.yaml:30]: github display of that is confusing =D

**Pull request closed without merging**